python/tskit/trees.py

```diff
@@ -5522,6 +5522,44 @@ def genetic_relatedness(
         :ref:`polarised <sec_stats_polarisation>`,
         and :ref:`return value <sec_stats_output_format>`.
 
+        What is computed depends on ``mode``:
+
+        "site"
+            Number of pairwise allelic matches in the window between two
+            sample sets relative to the rest of the sample sets. To be precise,
+            let `m(a,b)` denote the total number of pairwise allelic matches
+            between sample sets `a` and `b` and let `C` and `D` be
+            independently chosen sample sets. Then, this computes
+            `E[m(a,b) - m(a,C) - m(b,D) + m(C,D)]`.
+            This can also be seen as the covariance of a quantitative trait
+            corresponding to two sample sets (or "individuals"). Let each
```

Comment:
how about "the covariance of a quantitative trait determined by additive contributions from the genomes in each sample set."

Reply:
(since the "or individuals") might lead people to think you can pass in tskit-individuals here.